ananse/peakpredictor.py

```diff
@@ -83,8 +88,15 @@ def __init__(
         self.pfmfile = pfmfile
         # load motifs, f2m and motif_graph
         self._load_motifs(factors=factors)
+
+        # load CAGE bidirectional sites data and regions
+        if cage_tpms is not None:
+            if _istable(cage_tpms):
```

Comment:
ATAC/ChIP-seq input can be a table or a (list of) bam file(s). Function `_istable()` checks which input we get.

`cage_tpms` is always a table, so (unless I'm missing something) you can drop line `if _istable(cage_tpms):`